logstash-core/src/main/java/org/logstash/RubyUtil.java

```diff
@@ -446,6 +449,10 @@
         SLOW_LOGGER = loggingModule.defineClassUnder(
             "SlowLogger", RUBY.getObject(), SlowLoggerExt::new);
         SLOW_LOGGER.defineAnnotatedMethods(SlowLoggerExt.class);
+        DEPRECATION_LOGGER = loggingModule.defineClassUnder("DeprecationLogger", RUBY.getObject(),
+                                                            DeprecationLoggerExt::new);
```

Comment:
cosmetic: for multi line arguments we seems to put them on their own lines and ident from the left, I suggest we keep a consistent formatting.